### ISSUE TITLE
Add Bluesky handle of R-Ladies

### ISSUE THREAD

**Issue by cosimameyer** [rladies/awesome-rladies-blogs#97]
@drmowinckels @SoyAndrea I hope you're doing well ☺️ 

I'm working on a Bluesky integration of the R-Ladies' bot and wondered if you'd be OK with adding `"bluesky": "@handle.bsky.social"` as a new key-value pair to the JSON files?)

If yes, I'll start reaching out to those that I can find over there individually and open PRs to ask for their consent.

**Comment by drmowinckels** [comment]
go for it! I've also added bluesky as possible social media in the directory form. Need to integrate it with the website too, but this is great!

**Comment by drmowinckels** [comment]
if you are reaching out individually, can you also ask if they are ok with being in the R-Ladies starter pack?

**Comment by cosimameyer** [comment]
Awesome! I'll go ahead (hopefully do it on Sunday) and reach out to them ☺️ will also ask about the starter pack - is it best if I collect all answers and share them with you later? Or should I tag you in the PRs? Whatever you prefer is good for me as well ☺️

**Comment by cosimameyer** [comment]
@drmowinckels would you be ok featuring your R-Ladies starter packin the Awesome list? https://github.com/ericclemmons/awesome-starter-packs

**Comment by cosimameyer** [comment]
@drmowinckels All the PR referred here agreed to be added to the R-Ladies starter pack on GitHub :)

**Comment by ivelasq** [comment]
Hi @cosimameyer! I am OK with being added to a Bluesky starter pack, thanks for asking 😊

**Comment by drmowinckels** [comment]
> @drmowinckels would you be ok featuring your R-Ladies starter pack in the Awesome list? https://github.com/ericclemmons/awesome-starter-packs

yes ofcourse! But! I think its better to add the R-Ladies one (I am currently adding new members to both, untill the communications team is ready to start announcing on Bluesky). https://go.bsky.app/3y5RUfs
Once they announce it, I will edit my starter pack description pointing to that one in stead.

**Comment by cosimameyer** [comment]
> > @drmowinckels would you be ok featuring your R-Ladies starter pack in the Awesome list? https://github.com/ericclemmons/awesome-starter-packs
> 
> yes ofcourse! But! I think its better to add the R-Ladies one (I am currently adding new members to both, untill the communications team is ready to start announcing on Bluesky). https://go.bsky.app/3y5RUfs Once they announce it, I will edit my starter pack description pointing to that one in stead.

Okidoki, I'll open a PR and link to the R-Ladies starter pack by R-Ladies :)
